modules/opensearch/opensearch.client.inc

```diff
@@ -532,6 +558,40 @@ function _opensearch_cache_insert(TingClientRequest $request, $response, array &
   }
 }
 
+/**
+ * Retrieve a earch response based on a request.
```

Comment:
nitpicking: missing "s" in search in the function comment.

Reply:
This DocBlock has been removed in a subsequent commit so I do not think it is worth fixing.